src/spinner/index.ts

```diff
@@ -100,7 +102,6 @@ async function gradient(
     stop() {
       done = true;
       stdin.removeListener("keypress", keypress);
-      clearInterval(interval);
```

Comment:
Why was this removed? I’d expect it to be required still.

Reply:
The `interval` property was only initialized as `let interval: NodeJS.Timeout;` and then never used anywhere.
I'm assuming this was just a leftover of some old code that got removed.
TypeScript was complaining about it, I'm assuming the older version of TypeScript was slightly more lenient on it.